### ISSUE TITLE
Recopilar información del sitio web actual

### ISSUE THREAD

**Issue by zomars** [Maqueteros/bomberosculiacan#1]
Necesitamos recopilar la información que tienen en sitio web actual:

http://bomberosculiacan.org.mx/

**Comment by marioecg** [comment]
Yo la agrego en google drive, solo que no tengo permisos para crear o editar archivos.

**Comment by Aldaz08** [comment]
Ya hiciste algo @marioecg? para no hacer cosas similares

**Comment by marioecg** [comment]
@Aldaz08 SÍ, llevo la página de inicio, organigrama, galería y logros, también haré la de contacto. Si puedes apóyame con la de noticias y recomendaciones.

**Comment by zomars** [comment]
Te asigno la tarea entonces @marioecg

**Comment by Aldaz08** [comment]
si, dónde lo estas poniendo?

**Comment by zomars** [comment]
Tenemos una carpeta acá: https://drive.google.com/drive/folders/1tx5t8LsxjNravoVdfx1xKSe32ryyFU6q?usp=sharing

**Comment by marioecg** [comment]
@Aldaz08 estoy agregando el contenido dentro de la carpeta "Información sitio web actual"  en el archivo de "Información". Decidí separar las imágenes en otra carpeta si se llega a reutilizar algo, porque en el documento de texto me desparramaba todo 😅.

**Comment by Aldaz08** [comment]
sale!

**Comment by Aldaz08** [comment]
@marioecg  Hice una tabla general del contenido dividida por niveles de profundidad, para tener un panorama de como podemos reestructurar, lo pondré al final de tu doc.

**Comment by marioecg** [comment]
Me parece perfecto @Aldaz08.

**Comment by Aldaz08** [comment]
Crees que sea importante poner los posts que tienen ahorita? en el doc @marioecg

**Comment by marioecg** [comment]
@Aldaz08 yo pienso que sí para saber la que tan importantes son y ver si se van a reutilizar en el nuevo sitio. En total creo que son menos de 20. O como la vez?

**Comment by zomars** [comment]
@Aldaz08 considero importante recopilar todo lo que tienen, y a partir de ahi, generar un nuevo documento con la info nueva ya procesada. Nos sirve tambien para tener un antes y un despues.

**Comment by Aldaz08** [comment]
arre!

**Comment by Aldaz08** [comment]
No tengo permisos para cargar archivos en la carpeta imagenes @zomars ¿Me puedes dar permisos?

**Comment by zomars** [comment]
@Aldaz08 listo!

**Comment by zomars** [comment]
Vamos a mover este proyecto al colectivo @cahita
